warehouse/query-core/src/main/java/datawave/query/planner/DefaultQueryPlanner.java

```diff
@@ -963,7 +970,10 @@ protected ASTJexlScript updateQueryTree(ScannerFactory scannerFactory, MetadataH
         // groom the query so that any nodes with the literal on the left and the identifier on
         // the right will be re-ordered to simplify subsequent processing
 
-        config.setQueryTree(timedInvertSwappedNodes(timers, config.getQueryTree()));
+        {
```

Comment:
just wondering, were the braces here meant to block out the method call?

Reply:
No, not sure how that got left in there. Removing it.

Reply:
Now removed.